src/bin/lpython.cpp

```diff
@@ -222,7 +221,7 @@ int emit_cpp(const std::string &infile,
     std::cout << res.result;
     return 0;
 }
-
+#define HAVE_LFORTRAN_LLVM 1
```

Comment:
I think this should not be here, this is defined by cmake in `config.h`.

Reply:
yes, you're right, i just put it there just to test something and i forget to erasr it,sorry.